src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { FiMenu } from 'react-icons/fi';
+import { Route, Routes, useLocation } from 'react-router-dom';
+import Welcome from './components/Welcome';
+import HotelRooms from './components/HotelRooms';
+import ReservationsList from './components/ReservationsList';
+import DeleteHotel from './components/DeleteHotel';
+import ReserveForm from './components/ReserveForm';
+import AddHotel from './components/AddHotel';
+import SideBar from './components/SideBar';
+import Authentication from './components/Authentication';
+import Details from './components/Details';
+import { toggleNav } from './redux/sideBar/sideBarSlice';
+
+import './App.css';
+
+const App = () => {
```

Comment:
- [ ] When I create a new hotel room, it gets duplicated in the desktop version carrousel:

![image](https://github.com/Jolak5/hotel-booking-app-frontend/assets/57421823/dd06e435-1bca-4452-8dd7-ddd0039a1386)

This is because the carrousel has been set up to cover any empty space, so it duplicates the items.

Kindly make it so no item gets duplicated.

- [ ] In "My reservations" the items are getting duplicated as well.

I have only 1 reservation for test room 1, but it appears twice:

![image](https://github.com/Jolak5/hotel-booking-app-frontend/assets/57421823/0c7717e9-a010-4035-a77d-70fb02d55e87)

Kindly, make it so no item gets duplicated.

_note:  Both of these issues only happen in the desktop version._